integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -867,3 +868,99 @@ def test_subtraction_overflow_with_ansi_enabled(data, tp, expr):
+def test_abs_ansi_no_overflow_day_time_interval():
+    DAY_TIME_GEN_NO_OVER_FLOW = DayTimeIntervalGen(min_value=timedelta(days=-2000*365), max_value=timedelta(days=3000*365))
+    assert_gpu_and_cpu_are_equal_collect(
+        lambda spark: unary_op_df(spark, DAY_TIME_GEN_NO_OVER_FLOW).selectExpr('abs(a)'),
+        conf=ansi_enabled_conf)
+
+@pytest.mark.skipif(is_before_spark_330(), reason='DayTimeInterval is not supported before Pyspark 3.3.0')
+def test_abs_ansi_overflow_day_time_interval():
+    assert_gpu_and_cpu_error(
+        df_fun=lambda spark: _get_overflow_df(spark, [timedelta(microseconds=LONG_MIN)], DayTimeIntervalType(), 'abs(a)').collect(),
+        conf=ansi_enabled_conf,
+        error_message='ArithmeticException')
+
+@pytest.mark.skipif(is_before_spark_330(), reason='DayTimeInterval is not supported before Pyspark 3.3.0')
+def test_addition_day_time_interval():
+    DAY_TIME_GEN_NO_OVER_FLOW = DayTimeIntervalGen(min_value=timedelta(days=-2000*365), max_value=timedelta(days=3000*365))
+    assert_gpu_and_cpu_are_equal_collect(
+        lambda spark: two_col_df(spark, DAY_TIME_GEN_NO_OVER_FLOW, DAY_TIME_GEN_NO_OVER_FLOW).select(
+            f.col('a') + f.col('b')))
+
+@pytest.mark.skipif(is_before_spark_330(), reason='DayTimeInterval is not supported before Pyspark 3.3.0')
+def test_addition_ansi_no_overflow_day_time_interval():
+    DAY_TIME_GEN_NO_OVER_FLOW = DayTimeIntervalGen(min_value=timedelta(days=-2000*365), max_value=timedelta(days=3000*365))
+    assert_gpu_and_cpu_are_equal_collect(
+        lambda spark: two_col_df(spark, DAY_TIME_GEN_NO_OVER_FLOW, DAY_TIME_GEN_NO_OVER_FLOW).select(
+            f.col('a') + f.col('b')),
+        conf=ansi_enabled_conf)
+
+@pytest.mark.skipif(is_before_spark_330(), reason='DayTimeInterval is not supported before Pyspark 3.3.0')
+def test_add_overflow_with_ansi_enabled_day_time_interval():
+    assert_gpu_and_cpu_error(
+        df_fun=lambda spark: spark.createDataFrame(
+            SparkContext.getOrCreate().parallelize([(timedelta(microseconds=LONG_MAX), timedelta(microseconds=10)),]),
+            StructType([StructField('a', DayTimeIntervalType()), StructField('b', DayTimeIntervalType())])
+        ).selectExpr('a + b').collect(),
+        conf=ansi_enabled_conf,
+        error_message='ArithmeticException')
+
+@pytest.mark.skipif(is_before_spark_330(), reason='DayTimeInterval is not supported before Pyspark 3.3.0')
+def test_subtraction_day_time_interval():
+    DAY_TIME_GEN_NO_OVER_FLOW = DayTimeIntervalGen(min_value=timedelta(days=-2000*365), max_value=timedelta(days=3000*365))
+    assert_gpu_and_cpu_are_equal_collect(
+        lambda spark: two_col_df(spark, DAY_TIME_GEN_NO_OVER_FLOW, DAY_TIME_GEN_NO_OVER_FLOW).select(
+            f.col('a') - f.col('b')))
+
+@pytest.mark.skipif(is_before_spark_330(), reason='DayTimeInterval is not supported before Pyspark 3.3.0')
+def test_subtraction_ansi_no_overflow_day_time_interval():
```

Comment:
This can be merged with `test_subtraction_day_time_interval` by parameterizing the ansi config.

Reply:
done